applications/management/commands/expire_applications.py

```diff
@@ -4,36 +4,40 @@
 from django.core.management.base import BaseCommand
 from django.utils import timezone
 
-
+from app.hackathon_variables import APPLICATION_EXPIRATION_TYPES
 from applications import models, emails
+from applications.views import VIEW_APPLICATION_TYPE
 
 
 class Command(BaseCommand):
     help = 'Checks invites that have expired and sends reminders 24 before'
 
     def handle(self, *args, **options):
         fourdaysago = timezone.now() - timedelta(days=4)
+        msgs = []
         self.stdout.write('Checking reminders...')
-        reminders = models.Application.objects.filter(
-            status_update_date__lte=fourdaysago, status=models.APP_INVITED)
-        self.stdout.write('Checking reminders...%s found' % reminders.count())
+        for (type_app, do) in APPLICATION_EXPIRATION_TYPES:
+            if do:
```

Comment:
Write to stdout to say that you are skipping X type of user.

Reply:
Done